Casks/s/swiftdialog.rb

```diff
@@ -0,0 +1,32 @@
+cask "swiftdialog" do
+  version "2.5.6,4805"
+  sha256 "69bd7f6412a1b2393906fc059369e65c5ae58af5cc8d289467effd11d8862b2a"
+
+  url "https://github.com/swiftDialog/swiftDialog/releases/download/v#{version.csv.first}/dialog-#{version.csv.first}-#{version.csv.second}.pkg",
+      verified: "github.com/swiftDialog/swiftDialog/"
+  name "swiftDialog"
+  desc "Admin utility that presents custom dialogs or messages from shell scripts"
+  homepage "https://swiftdialog.app/"
+
+  livecheck do
+    url "https://api.github.com/repos/swiftDialog/swiftDialog/releases/latest"
+    regex(/dialog[._-](\d+(?:\.\d+)+)[._-](\d+)\.pkg/i)
+    strategy :page_match do |page, regex|
+      match = page.match(regex)
+      next if match.blank?
+
+      "#{match[1]},#{match[2]}"
+    end
```

Comment:
We tend to get rate-limited by using API urls directly, so the below should provide an authenticated way of achieving the same thing.

```suggestion
    url :url
    regex(/dialog[._-](\d+(?:\.\d+)+)[._-](\d+)\.pkg/i)
    strategy :github_latest do |json, regex|
      json["assets"]&.map do |asset|
        match = asset["browser_download_url"]&.match(regex)
        next if match.blank?

        "#{match[1]},#{match[2]}"
      end
```

Reply:
thanks 🙂 

I've merged these in but checks still seem to be failing. I'll pull down the changes and go through the cask to see what else is going on.

Reply:
Sorry that was my mistake! It's hard to match up the `end` blocks in comment suggestions sometimes. I'll add another suggestion now.